client/my-sites/plugins/plugin-install-button/index.jsx

```diff
@@ -233,17 +236,30 @@ export class PluginInstallButton extends Component {
 		);
 	}
 
-	render() {
-		const { selectedSite } = this.props;
-		if ( selectedSite.unreachable ) {
+	renderNoticeOrButton() {
+		const { selectedSite, siteIsConnected } = this.props;
+
+		if ( siteIsConnected === false ) {
 			return this.renderUnreachableNotice();
 		}
+
 		if ( ! selectedSite.canUpdateFiles ) {
 			return this.renderDisabledNotice();
 		}
 
 		return this.renderButton();
 	}
+
+	render() {
+		const { siteId } = this.props;
+
+		return (
+			<div>
+				<QuerySiteConnectionStatus siteId={ siteId } />
```

Comment:
Let's also update QuerySiteConnectionStatus with 2 quick changes:
1. Remove the isRequired from the siteId prop https://github.com/Automattic/wp-calypso/blob/master/client/components/data/query-site-connection-status/index.jsx#L15
2. Do an early return if siteId is falsey: https://github.com/Automattic/wp-calypso/blob/master/client/components/data/query-site-connection-status/index.jsx#L31

This is good to 🚢 after these changes are done